packages/suite/src/utils/suite/guide.ts

```diff
@@ -0,0 +1,40 @@
+import { GuideCategoryType, GuideItemType } from '@suite-types/guide';
+
+export const getItemTitle = (item: GuideItemType, language: string): string =>
+    item.title[language || 'en'];
+
+export const getNodeById = (id: string, root: GuideItemType): GuideItemType | undefined => {
+    if (id === root.id) {
+        return root;
+    }
+
+    if (root.type === 'page') {
```

Comment:
I know this is semantically identical (as long as we only have pages and categories) but let's write the condition so that it corresponds the the actual intention
```suggestion
     if (root.type !== 'category') {

Reply:
Resolved.